src/App.js

```diff
@@ -101,10 +101,95 @@ class Board extends React.Component {
     return true;
   }
 
+  //Remove a random number from the board
+  removeRandomNumber(valueBoard) {
+
+    //Pick a random position o the board
+    let randomPositionOfBoard = Math.floor(Math.random() * valueBoard.length);
+
+    let backup = [];
+
+    //Change the random number if the position it's on was a zero
+    while(parseInt(valueBoard[randomPositionOfBoard]) === 0) {
+      //Reset random number
+      randomPositionOfBoard = Math.floor(Math.random() * valueBoard.length);
+    }
+
+    //Back up position of board's value removed and it's number
+    backup.push(randomPositionOfBoard);
+    backup.push(valueBoard[randomPositionOfBoard]);
+
+    //Set value of the random picked position to zero
+    valueBoard[randomPositionOfBoard] = 0;
+    
+    //Tell which position got their number removed and what number
+    return backup;
+  }
+
+  //Undo removeRandomNumber
+  undoRemoveRandomNumber(undo, valueBoard) {
+    //Reset the position, given from removeRandomNumber method, of the board's value
+    valueBoard[undo[0]] = undo[1];
+  }
+
+  findAllPossibleAnswers(valueBoard, backup) {
+    let allPossibleAnswers = [];
+    let oneSetOfAnswers = [];
+    for(let i = 0; i < backup.length; i++) {
+      for(let j = 1; j < 10; j++) {
+        if( this.isPossibleRow(j, this.returnRow(backup[i][0]), valueBoard) &&
+            this.isPossibleCol(j, this.returnCol(backup[i][0]), valueBoard) &&
+            this.isPossibleBlock(j, this.returnBlock(backup[i][0]), valueBoard)
+        ) {
+          oneSetOfAnswers.push(j);
+        }
+      }
+      allPossibleAnswers.push([backup[i][0], oneSetOfAnswers]);
+      oneSetOfAnswers = [];
+    }
+    return allPossibleAnswers;
+  }
+
+  /* In Progress */
```

Comment:
Typically if something is in progress/not really working we don't want to include it in the PR.

Reply:
Even if it doesn't affect the code?